setup.py

```diff
@@ -102,9 +102,9 @@ def _get_cmdclass():
 
 def spirv_compile():
     if IS_LIN:
-        compiler = "clang"
+        compiler = "icx"
     if IS_WIN:
-        compiler = "clang.exe"
+        compiler = "icx.exe"
```

Comment:
@reazulhoque is the Windows executable called `icx.exe`? CI seems to be dead, so is there any other way to verify?

Reply:
Verified, it is there.